### ISSUE TITLE
exit with non-zero value when code is not formatted

### ISSUE THREAD

**Issue by x4m3** [jidicula/clang-format-action#1]
hi, thanks for the action, it works great.

just a question, do you know how i can make my job fail if a file is not formatted correctly?

i tried this but it doesn't seem to work:
```yml

  check-code:
    needs: build
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v2
    - name: run clang-format
      uses: jidicula/clang-format-action@master
    - name: The job has failed
      if: ${{ failure() }}
      run: echo "code is not clean"; exit 1
```

do you happen to know i could make that work?

thank you

**Comment by jidicula** [comment]
Hey,

Thanks so much for trying out my action! I had just made it as a first foray into this for the GitHub Actions Hackathon a few weeks ago and I wasn't expecting it to be used by anyone else 😛.

I don't think the issue you were reporting had anything to do with the YAML creation when adding the action to your repo – instead it was an issue with `entrypoint.sh`. I found the source of the bug and fixed it in #2. After testing on a couple of my own private C repos, I think this issue has been resolved.

Let me know if you still have any issues with it and I'll see what I can do. Thank you again for catching this!
